superset/reports/commands/execute.py

```diff
@@ -132,7 +136,7 @@ def create_log(
             end_dttm=datetime.utcnow(),
             value=self._report_schedule.last_value,
             value_row_json=self._report_schedule.last_value_row_json,
-            state=state,
+            state=self._report_schedule.state,
```

Comment:
There's no need to pass in the state given the report schedule record has been updated with the new state.